lib/dialects/mssql/query-generator.js

```diff
@@ -27,17 +25,17 @@ module.exports = (function() {
       return [
         'SELECT "name" as "schema_name" FROM sys.schemas as s',
         'WHERE "s"."name" NOT IN (',
-          "'INFORMATION_SCHEMA', 'dbo', 'guest', 'sys', 'archive'",
-        ")", "AND", '"s"."name" NOT LIKE', "'db_%'"
+          '\'INFORMATION_SCHEMA\', \'dbo\', \'guest\', \'sys\', \'archive\'',
+        ')', 'AND', '"s"."name" NOT LIKE', '\'db_%\''
       ].join(' ');
     },
 
     versionQuery: function() {
-      return "SELECT @@VERSION as 'version'";
+      return 'SELECT @@VERSION as \'version\'';
```

Comment:
Would you prefere to ignore the errors or remove the quotmark in general?
I ignored quite a few of these errors in the tests.